tox.ini

```diff
@@ -1066,18 +1070,25 @@ changedir = {toxinidir}/packages
 allowlist_externals =
     docker
     bash
+    env
 setenv =
     PIP_PACKAGES = {env:PIP_PACKAGES:llama-index opendp}
     SYSTEM_PACKAGES = {env:SYSTEM_PACKAGES:curl wget}
+    BUILD_PLATFORM = {env:BUILD_PLATFORM:linux/amd64}
```

Comment:
Q: Why have used explicitly linux/amd64, as it would slow the build on arm 64 based systems?

Reply:
Oh this is just to make sure if I want to test this locally on a different platform, i can pass in a buildx platform through env var.

Reply:
Also I moved to buildx build - so it needs a platform arg which defaults to linux/amd64